packages/experience-builder-sdk/src/core/preview/assemblyUtils.spec.ts

```diff
@@ -23,7 +23,11 @@ describe('resolveAssembly', () => {
       locale: 'en-US',
     });
 
-    const result = resolveAssembly({ node: containerNode, entityStore });
+    const result = resolveAssembly({
+      node: containerNode,
+      entityStore,
+      parentPatternProperties: {},
```

Comment:
A test or two that specify pattern property inputs might be nice but since it's just a straightforward merge you can just use your best judgement there.